src/main/java/meteordevelopment/meteorclient/renderer/Renderer2D.java

```diff
@@ -28,7 +28,7 @@ public Renderer2D(boolean texture) {
         lines = new ShaderMesh(Shaders.POS_COLOR, DrawMode.Lines, Mesh.Attrib.Vec2, Mesh.Attrib.Color);
     }
 
-    @Init(stage = InitStage.Pre)
+    @Init(stage = InitStage.Pre, dependencies = Shaders.class)
```

Comment:
It was just never an issue, because dependencies were never used up until now, but Renderer2D definitely depends on Shaders, I probably missed some other init functions that have dependencies, but I guess it works for now